tests/network/test_management_disable_address_type.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+# Requirements:
+# - one XCP-ng host (--host) (>= 8.3 for IPv6 test)
```

Comment:
Should it require the fixture which skips tests when the version is not appropriate?

Reply:
Fixed by only having on test we'll run on 2 different hosts.